app/scripts/models/auth_brokers/fx-desktop-v2.js

```diff
@@ -64,6 +65,24 @@ define(function (require, exports, module) {
         .then(function () {
           return proto.afterCompleteResetPassword.call(self, account);
         });
+    },
+
+    fetch: function () {
+      var self = this;
+      return proto.fetch.call(self)
+        .then(function () {
+          if (! self.environment.isAboutAccounts()) {
+            // The default behavior of FxDesktop brokers is to halt before
+            // the signup confirmation poll because about:accounts takes care
+            // of polling and updating the UI. However if we are not in about:accounts
+            // we do not want the halting behavior.
+            Object.keys(self._behaviors.attributes).forEach(function (behavior) {
+              if (self._behaviors.attributes[behavior].type === 'halt') {
+                self.setBehavior(behavior, new NullBehavior());
+              }
```

Comment:
Yeah, I was wondering how we could do introspection on the behavior, this seems reasonable.